UnityGLTF/Assets/UnityGLTF/Scripts/UniformMaps/MetalRoughMap.cs

```diff
@@ -57,5 +57,15 @@ public override IUniformMap Clone()
 			base.Copy(copy);
 			return copy;
 		}
+		
+		public override double RoughnessFactor
```

Comment:
Shouldn't this be of type `float`?

Reply:
My proposal is inspired by the definition of MetallicFactor in class MetalRough2StandardMap, where you can find the same mix of ```double``` and ```float```:

```
public virtual double MetallicFactor
{
	get { return _material.GetFloat("_Metallic"); }
	set { _material.SetFloat("_Metallic", (float)value); }
}
```
However, you are right, on a second look this seems a bit odd, but I don't know if it has any impact in practice (or maybe is even done in this way by intention).

The same can also be found in other classes, e.g. SpecGloss2StandardMap:

```
public virtual double GlossinessFactor
{
	get { return _material.GetFloat("_GlossMapScale"); }
	set
	{
		_material.SetFloat("_GlossMapScale", (float)value);
		_material.SetFloat("_Glossiness", (float)value);
	}
}
```

Reply:
Given that the materials only support single-precision uniforms, these should all probably be floats and not doubles.